packages/enclave-portal/src/features/enclave/client.ts

```diff
@@ -0,0 +1,530 @@
+/*****************************************
+ * Portal representation of cohort filters
+ *****************************************/
+interface IncludesOperation {
+  readonly operator: "includes";
+  readonly field: string;
+  readonly operands: ReadonlyArray<string>;
+}
+
+interface GreaterThanOEqualsThanOperation {
```

Comment:
```suggestion
interface GreaterThanOrEqualsThanOperation {
```
Guessing it's a typo.